source/languages/en/riak/ops/running/handoff.md

```diff
@@ -263,7 +263,7 @@ Header | Description
 `Node` | The name of the node
 `Total` | Total number of active transfers throughout the entire cluster
 `Ownership` | Total number of ownership exchanges
-`Resize` | Total handoffs related to [[ring resizing]] operations
+`Resize` | Total handoffs related to ring resizing operations
```

Comment:
`riak-admin handoff summary` does have an entry for Resize, so we probably want to leave it documented.
Maybe we want to add a note that the resize feature has been deprecated?